### PR TITLE
Minor Dockerfile fix

### PR DESCRIPTION
**Pull request by jason-fox**
* Save `Dockerfile` as UTF-8 - to correct the `MAINTAINER`  Telefónica I+D
* Add `.dockerignore` file
* Use an `ARG` to set the node version

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+coverage/
+docs/
+ghpages/
+test/
+tools/
```
[the remark this answers is not in the document]
Comment (jason-fox) [reply .dockerignore]: Fixed - 97d940d

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
```
Comment (dcalvoalonso) [inline .dockerignore]: ´This folder will be removed with https://github.com/telefonicaid/iotagent-json/pull/339 so I would suggest not to include it.

Comment (jason-fox) [reply .dockerignore]: Fixed - 5bb8375 - `tools/` removed.

**Comment by dcalvoalonso** [comment]
LGTM

**Pull request merged**